CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## master / unreleased
+* [CHANGE] Ingester: Rename max-exemplars limits config to ingester.max-exemplars instead of blocks-storage.tsdb.max-exemplars. #5122
```

Comment:
Changelog should be fine. I think we can safely remove the entry as the previous has not been released.

Reply:
Removed.